catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/enumerations/ExperimentalEnumerationExtractor.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard.enumerations;
+
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+import com.google.common.collect.Sets;
+
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.validation.AttributeValidatorRegistry;
+import ddf.catalog.validation.violation.ValidationViolation;
+
+/**
+ * This class is Experimental and subject to change
+ */
+public class ExperimentalEnumerationExtractor {
```

Comment:
This is a really bad hack and we probably want to rip it all out first chance we get. So hopefully the only change we make to this one will be deleting it 🤞